stripe/src/main/java/com/stripe/android/paymentsheet/BasePaymentMethodsListFragment.kt

```diff
@@ -72,6 +72,8 @@ internal abstract class BasePaymentMethodsListFragment(
         paymentSelection: PaymentSelection,
         isClick: Boolean
     ) {
-        sheetViewModel.updateSelection(paymentSelection)
+        if (isClick) {
+            sheetViewModel.updateSelection(paymentSelection)
+        }
```

Comment:
Is there some scenario where we wanted `sheetViewModel.updateSelection(paymentSelection)` to fire without the click? I think this might apply after we set the default selection in the carousel, but I'm not sure.